tests/main.sh

```diff
@@ -21,14 +21,6 @@ assert ()
 	fi
 }
 validate () {
-	[ "$PY_VER" == "3.6" ] || \
-		assert "debugger available" "[ $($SHELL_CMD 'eval "$(cat)"' <<-END
-			pip list --format=freeze 2>/dev/null | \
-				grep ipykernel | \
-				grep -qv "ipykernel==5\." && \
-			echo done
-		END
-		) == 'done' ]" $LINENO
```

Comment:
Whoops. You accidentally removed a valid test that we were ignoring for the Python 3.6 case. We need to verify that the debugging feature is installed.